urls.py

```
@@ -0,0 +1,20 @@
+#!/usr/bin python3
+# -*- coding: utf-8 -*-
+"""
+@CreateTime    : 2024/5/6 14:30
+@Author  : 系统订单服务
+@file for: 订单模块URL配置
+"""
+
+from OrderManagement.views.order_view import order_blueprint
```

Pylint (3.3.7)

[error] Unable to import 'OrderManagement.views.order_view'

(E0401)